src/core/dicom-web-api.ts

```diff
@@ -127,3 +127,28 @@ export async function fetchInstanceThumbnail(
   const blob = new Blob([arrayBufferView], { type: 'image/jpeg' });
   return URL.createObjectURL(blob);
 }
+
+const LEVELS = ['series', 'studies'] as const;
+
+// takes a url like http://localhost:3000/dicom-web/studies/someid/series/anotherid
+// returns { host: 'http://localhost:3000/dicom-web', studies: 'someid', series: 'anotherid' }
+export function parseUrl(deepDicomWebUrl: string) {
+  // remove trailing slash
+  const sansSlash = deepDicomWebUrl.replace(/\/$/, '') ?? '';
```

Comment:
`replace` returns a string, so we don't need the `?? ''`.

Reply:
Fixed.